compiler/test/dotty/tools/dotc/CompilationTests.scala

```diff
@@ -172,7 +172,7 @@ class CompilationTests extends ParallelTesting {
       "tests/neg-custom-args/toplevel-samesource/S.scala",
       "tests/neg-custom-args/toplevel-samesource/nested/S.scala"),
       defaultOptions) +
-    compileFile("tests/neg-custom-args/i6300.scala", allowDeepSubtypes)
+    compileFile("tests/neg-custom-args/i6300.scala", allowDeepSubtypes) +
```

Comment:
Maybe move the + at the beginning of each line to make it more obvious. Though even better would be to just have a comma-separate list.

Reply:
Good idea. I will do some refactoring in another PR as this fix should be merged as soon as possible

Reply:
See #6427